plugins/out_stackdriver/stackdriver.c

```diff
@@ -455,20 +491,34 @@ static int stackdriver_format(struct flb_config *config,
 {
     int len;
     int array_size = 0;
+    /* The default value is 3: timestamp, jsonPayload, logName. */
+    int entry_size = 3; 
     size_t s;
     size_t off = 0;
     char path[PATH_MAX];
     char time_formatted[255];
     struct tm tm;
     struct flb_time tms;
-    severity_t severity;
     msgpack_object *obj;
     msgpack_unpacked result;
     msgpack_sbuffer mp_sbuf;
     msgpack_packer mp_pck;
     flb_sds_t out_buf;
     struct flb_stackdriver *ctx = plugin_context;
 
+    /* Parameters in severity */
+    bool severity_extracted = false;
```

Comment:
please use FLB_TRUE or FLB_FALSE for boolean values

Reply:
Done.